functions/Copy-DbaDatabase.ps1

```diff
@@ -995,10 +995,17 @@ function Copy-DbaDatabase {
 					}
 
 					$dbFinish = Get-Date
-
 					if ($NoRecovery -eq $false) {
+						$dbOwner = $sourceServer.Databases[$dbName].Owner
```

Comment:
This should be set to `$destServer.Databases[$dbName].Owner` shouldn't it?

This assumes the login exist on the destination. Pulling it from the source then `$dbOwner` will never be null so `Get-SaLoginName` will never be called.